man/systemd.netdev.xml

```diff
@@ -408,6 +408,13 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>VLANProtocol=</varname></term>
+        <listitem>
+          <para> Allows setting the protocol used for VLAN filtering. Takes 802.1q or 802.1ad as parameters.
+          </para>
```

Comment:
Takes one of .... Defaults to ?

Reply:
Please remove the space after `<para>`.

Use `<option>802.1Q</option>` and similar everywhere.

Break the line.

Reply:
Man need to be reworked @keszybz.